change/@ni-nimble-components-acb97d9b-ffbe-4f70-8c61-a42f00459247.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Reported issue with the toolbar buttons not being white on green in Storybook 8 - https://github.com/storybookjs/storybook/issues/26448

Reply:
I think for this PR  in .storybook/manager-head.html

We can add:

```html
<!--
Toolbar button color workaround
See: https://github.com/storybookjs/storybook/issues/26448
-->
<style>
  [data-test-id="sb-preview-toolbar"] button {
      color: white;
  }
</style>
```

Reply:
Oh that issue is resolved and the comments says it may have released Monday this week. So we might not need the manager-head workaround if we upgrade.

Reply:
Yep - looks resolved in the latest patch.